packages/summer-earn-protocol-subgraph/subgraph.yaml

```
@@ -168,7 +173,9 @@ templates:
         - name: DefaultReverseNameResolver
           file: ./abis/ens/default-resolver.json
         - name: GovernanceRewardsManager
-          file: ./abis/GovernanceRewardsManager.abi.json          
+          file: ./abis/GovernanceRewardsManager.abi.json   
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)